angular-ngrx-scss/src/app/repository/components/repo-header/repo-heading/repo-heading.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, Input } from '@angular/core';
+import { RepoState } from 'src/app/state/repository';
+
+@Component({
+  selector: 'app-repo-heading',
+  templateUrl: './repo-heading.component.html',
+  styleUrls: ['./repo-heading.component.scss'],
+})
+export class RepositoryHeadingComponent {
+  @Input()
+  repo?: RepoState;
+  @Input() owner = '';
+  @Input() name = '';
+
+  get ownerPath(): string {
+    return `/${this.owner}`;
+  }
+
+  get visibility(): string {
+    if (this.repo) {
+      return this.repo.visibility ? 'Private' : 'Public';
```

Comment:
suggestion: Visibility should be a string and not a boolean in the store - this getter should return the visibility value directly. Maybe something like:
```suggestion
      return this.repo.visibility || '';
```
So it'll default back to empty if that value isn't set, but otherwise returns the value?